epictrack-web/src/components/work/Dialog/index.tsx

```diff
@@ -35,14 +35,22 @@ export const WorkDialog = ({
     if (!workId) return;
     try {
       const response = await workService.getById(String(workId));
+
+      const fetchedWork = response.data;
+
+      // set simple_title to empty string if it is null
+      if (fetchedWork.simple_title === null) {
```

Comment:
Yes this is a fix, but I'm worried we're doing a lot. Maybe we can add.nullable() in the schema?

Reply:
We ideally shouldn't mutate objects like this, especially if they are const. We can perhaps do setWork({...fetchedWork, simple_title: fetchedWork.simple_title ?? ''}). This would create a new object rather than mutating and existing one

Reply:
Also we if simple_title is nullable we could perhaps check if adding .nullable() to the schema would work